### DISCUSSION TITLE
watchモードでのHTMLエラー表示

### DISCUSSION THREAD

**Discussion by koriym** [alps-asd/app-state-diagram#129] [Ideas]
見ているページはPHPからわからないわけで...これは技術的に難しいでしょうか？

**Comment by koriym** [comment]
生成した全てのページのHTMLを書き換えるとかでしょうか。

**Reply by NaokiTsuchiya** [reply]
gulpでエラー時にbrowsersyncにnotifyする例が以下です。参考になるかもしれないです。

https://gist.github.com/lmartins/cebf7d01992e18888ecf#file-gulpfile-js-L23-L26

**Reply by NaokiTsuchiya** [reply]
あくまでもnotifyなのでエラーを表示するという感じではなさそうですね

**Reply by koriym** [reply]
これでも良さそうですね
